.github/workflows/labeler.yml

```diff
@@ -38,41 +36,54 @@ jobs:
-      uses: elastic/assign-one-project-github-action@1.2.2
-      if: contains(steps.is_elastic_member.outputs.result, 'true') && github.event.pull_request
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+        configuration-path: .github/community-label.yml
+        enable-versioned-regex: 0
+
+    - name: Assign new internal pull requests to project in current iteration
+      if: (steps.checkUserMember.outputs.isTeamMember == 'true' || steps.checkUserMember.outputs.isExcluded == 'true') && github.event_name == 'pull_request'
+      run: |
+        project_id=$(gh api graphql -f query='query($project_number:Int!){ organization(login:"elastic"){ projectV2(number:$project_number) { id } } }' -F project_number=${PROJECT_NUMBER_ID} --jq '.data.organization.projectV2.id' )
+        item_id=$(gh project item-add ${PROJECT_NUMBER_ID} --url ${PULL_REQUEST_URL} --owner elastic --format json --jq .id)
+
+        status_field_json=$(gh api graphql -f query='query($project_id:ID!){ node(id:$project_id){ ... on ProjectV2 { fields(first:20) { nodes { ... on ProjectV2SingleSelectField { id name options { id name } } } } } } }' --jq '.data.node.fields.nodes[]|select(.name=="Status")' -F project_id=${project_id})
+        status_field_id=$( echo "$status_field_json" | jq -r '.id' )
+        status_field_value_id=$( echo "$status_field_json" | jq -r ".options[]|select(.name==\"${STATUS}\")|.id" )
+        gh project item-edit --id "${item_id}" --project-id "${project_id}" --field-id "${status_field_id}" --single-select-option-id "${status_field_value_id}"
+
+        agent_field_json=$(gh api graphql -f query='query($project_id:ID!){ node(id:$project_id){ ... on ProjectV2 { fields(first:20) { nodes { ... on ProjectV2SingleSelectField { id name options { id name } } } } } } }' --jq '.data.node.fields.nodes[]|select(.name=="Agent")' -F project_id=${project_id})
+        agent_field_id=$(echo "$agent_field_json" | jq -r '.id' )
+        agent_field_value_id=$(echo "$agent_field_json" | jq -r ".options[]|select(.name==\"${AGENT}\")|.id" )
+        gh project item-edit --id "${item_id}" --project-id "${project_id}" --field-id "${agent_field_id}" --single-select-option-id "${agent_field_value_id}"
+
+        iteration_field_json=$(gh api graphql -f query='query($project_id:ID!){ node(id:$project_id) { ... on ProjectV2 { fields(first: 20) { nodes { ... on ProjectV2IterationField { id name configuration { iterations { startDate id } } } } } } } }' --jq '.data.node.fields.nodes[] | select( .name == "Iteration" )' -F project_id=${project_id})
+        iteration_field_id=$(echo "$iteration_field_json" | jq -r '.id' )
+        iteration_field_value_id=$(echo "$iteration_field_json" | jq -r 'last(.configuration.iterations[]|select(.startDate<=(now|strftime("%Y-%m-%d"))))|.id')
+        gh project item-edit --id "${item_id}" --project-id "${project_id}" --field-id "${iteration_field_id}" --iteration-id "${iteration_field_value_id}"
```

Comment:
[for reviewer] this masterpiece of inlined graphql, jq and shell allows to avoid having to hardcode cryptic IDs of the project fields and their values. It should be reusable as-is in other repositories by changing the environment variables below.